install.bash

```diff
@@ -116,11 +119,15 @@ echo "***** INSTALLING FIFTYONE *****"
 if [ ${DEV_INSTALL} = true ] || [ ${VOXEL51_INSTALL} = true ]; then
     echo "Performing dev install"
     pip install -r requirements/dev.txt
-    pre-commit install
-    pip install -e .
+    if [ ${DEPS_ONLY} = false ]; then
```

Comment:
`install.bash` is meant to run on the host, with git and everything. Since it's not available in docker, avoid installing pre-commit hooks and installing from source. (The latter isn't needed anyways with PYTHONPATH set).